src/routes/spotify/Audio.svelte

```diff
@@ -60,7 +60,7 @@
   };
 
   const playPause = async (paused: boolean, src: string) => {
-    log(`playPause paused=${paused} src=${src}`);
+    log(`playPause deviceId=${deviceId} paused=${paused} src=${src}`);
```

Comment:
Prompt to play or not when there is something in the queue from previous session

Reply:
It doesn't prompt but it also doesn't auto play